core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -59,7 +75,7 @@
   private final List<ValidationNotice> validationNotices = new ArrayList<>();
   private final List<SystemError> systemErrors = new ArrayList<>();
   private final Map<String, Integer> noticesCountPerTypeAndSeverity = new HashMap<>();
-  private boolean hasValidationErrors = false;
+  private transient boolean hasValidationErrors = false;
```

Comment:
Do you need `transient` here? I believe that would only apply if you were serializing `NoticeContainer` itself.

Reply:
Just flagging this again - do you need `transient` here?

Reply:
Oops, indeed, this is not needed.